Source/AST/Stylers/Styler.swift

```diff
@@ -15,6 +15,13 @@ import Foundation
 /// A styler is used in conjunction with an instance of `AttributedStringVisitor` in order
 /// to generate an NSAttributedString from an abstract syntax tree.
 public protocol Styler {
+
+    #if os(Linux)
+    typealias PrefixAttributeKey = NSAttributedStringKey
```

Comment:
Official Swift 5 Docker image still has compiler which used `NSAttributedStringKey` instead of `NSAttributedString.Key`.